Source/stream.ts

```diff
@@ -278,6 +278,15 @@ const FOX_PLUS: StreamService = {
   ]
 };
 
+const FRIDAY: StreamService = {
+  name: 'friDay Video',
+  rules: [
+    'DOMAIN-SUFFIX,fetnet.net',
```

Comment:
`fetnet.net` doesn't really have anything to go with "Friday".

Fetnet is a Taiwanese ISP that happens to run "Friday", and that's all.

Please replace this with a more specific domain suffix rule.

Reply:
Also, please provide evidence that this domain can only be accessed through Taiwan IPs.